<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WizardSupplement</class>
 <widget class="QWizard" name="WizardSupplement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wizard</string>
  </property>
  <widget class="QWizardPage" name="wizardPage1">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Type</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="w_nom"/>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="w_type">
      <item>
       <property name="text">
        <string>Background</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Bestiaire</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Extension</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sc√©nario</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="l_langue">
      <property name="text">
       <string>Langue</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="w_langue">
      <item>
       <property name="text">
        <string>VF</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>VO</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="l_support">
      <property name="text">
       <string>Support</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="w_support">
      <item>
       <property name="text">
        <string>Livre</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DownLoad</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
